filament/src/RenderPass.cpp

```diff
@@ -199,6 +199,12 @@ void RenderPass::setupColorCommand(Command& cmdDraw,
 
     cmdDraw.key = isBlendingCommand ? keyBlending : keyDraw;
     cmdDraw.primitive.rasterState = ma->getRasterState();
+    if (!isBlendingCommand && Variant::isSSRVariant(variant)) {
```

Comment:
Is the compiler smart enough to get rid of the branch? Should we instead put the condition in the assignment with bit masking?

Reply:
okay, made it more branchless.